demo/midgard-node/src/services/always-succeeds.ts

```diff
@@ -62,29 +54,74 @@ export const makeAuthenticatedValidator = (
 
 const makeAlwaysSucceedsService: Effect.Effect<
   {
+    hubOracleAuthValidator: AuthenticatedValidator;
     stateQueueAuthValidator: AuthenticatedValidator;
+    registeredOperatorsAuthValidator: AuthenticatedValidator;
+    activeOperatorsAuthValidator: AuthenticatedValidator;
+    schedulerAuthValidator: AuthenticatedValidator;
+    retiredOperatorsAuthValidator: AuthenticatedValidator;
+    escapeHatchAuthValidator: AuthenticatedValidator;
+    fraudProofCatalogueAuthValidator: AuthenticatedValidator;
+    fraudProofAuthValidator: AuthenticatedValidator;
     depositAuthValidator: AuthenticatedValidator;
   },
   never,
   NodeConfig
 > = Effect.gen(function* () {
-  return yield* Effect.gen(function* () {
-    const stateQueueAuthValidator = yield* makeAuthenticatedValidator(
-      "always_succeeds.state_queue_spend.else",
-      "always_succeeds.state_queue_mint.else",
-    );
-
-    const depositAuthValidator = yield* makeAuthenticatedValidator(
-      "always_succeeds.deposit_spend.else",
-      "always_succeeds.deposit_mint.else",
-    );
+  const hubOracleAuthValidator = yield* makeAuthenticatedValidator(
+    "always_succeeds.hub_oracle_spend.else",
+    "always_succeeds.hub_oracle_mint.else",
+  );
+  const schedulerAuthValidator = yield* makeAuthenticatedValidator(
+    "always_succeeds.scheduler_spend.else",
+    "always_succeeds.scheduler_mint.else",
+  );
```

Comment:
Consider delegating more logic to the helper. We might be able to have less duplications:
```suggestion
  const schedulerAuthValidator = yield* makeAuthenticatedValidator("scheduler");
```